### PR TITLE
CASMNET-1412 - Add ExternalDNS A record changes and PowerDNS deprecation notice in 1.2 release notes

### PR DESCRIPTION
**Pull request by spillerc-hpe**
## Summary and Scope

Restructure PowerDNS documentation and document bifurcated CAN URL moves that are not backwards compatible in the CSM 1.2 release notes.

## Issues and Related PRs

* Resolves [CASMNET-1412](https://jira-pro.its.hpecorp.net:8443/browse/CASMNET-1412)
* Change will also be needed in `release/1.2` and `release/1.2.5`

## Pull Request Checklist

- [ ] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [x] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -18,7 +18,28 @@ CSM 1.2 contains approximately 2000 changes spanning bug fixes, new feature deve
...
+  | auth.shasta.dev.cray.com         | auth.cmn.shasta.dev.cray.com            |
+  | nexus.shasta.dev.cray.com        | nexus.cmn.shasta.dev.cray.com           |
+  | grafana.shasta.dev.cray.com      | grafana.cmn.shasta.dev.cray.com         |
+  | prometheus.shasta.dev.cray.com   | prometheus.cmn.shasta.dev.cray.com      |
+  | alertmanager.shasta.dev.cray.com | alertmanager.cmn.shasta.dev.cray.com    |
+  | vcs.shasta.dev.cray.com          | vcs.shasta.dev.cray.com                 |
+  | kiali-istio.shasta.dev.cray.com  | kiali-istio.shasta.dev.cray.com         |
+  | s3.shasta.dev.cray.com           | s3.shasta.dev.cray.com                  |
+  | sma-grafana.shasta.dev.cray.com  | sma-grafana.shasta.dev.cray.com         |
+  | sma-kibana.shasta.dev.cray.com   | sma-kibana.shasta.dev.cray.com          |
```
Comment (johren-hpe) [inline RELEASE_NOTES.md:37]: missing the cmn in right column for vcs, kiali-istio, s3, sma-grafana, and sma-kibana.

**Review by johren-hpe — CHANGES_REQUESTED** [review]

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -18,7 +18,28 @@ CSM 1.2 contains approximately 2000 changes spanning bug fixes, new feature deve
 * Release Cray Automated Network Utility - CANU V1.0.0
 * Performance improvements to Unbound and DHCP Helper
 * Initial Release of Bifurcated CAN
-* PowerDNS including PowerDNS Manager and Troubleshooting Guide
+
+  The user and administrative traffic segregation introduced by Bifurcated CAN has changed the URLs for certain services as it is now necessary to include the network path in the fully qualified domain name.
```
Comment (johren-hpe) [inline RELEASE_NOTES.md:22]: May want to be specific that only the API service is available on all both CMN and CAN/CHN.   The rest are CMN-only.   It should be obvious from the examples, but just to be crystal clear.

**Review by johren-hpe — CHANGES_REQUESTED** [review]

File: RELEASE_NOTES.md (hunk 2 above)
Comment (spillerc-hpe) [reply RELEASE_NOTES.md]: Added language to that effect.

**Review by dle-hpe — APPROVED** [review]

**Comment by mharding-hpe** [comment]
Note that this PR is now against the new main branch (which began its life as release/1.2). Ideally you should rebase, and then cherry-pick the commit into the older release branches (release/1.2 and 1.2.5).

File: RELEASE_NOTES.md (hunk 1 above)
Comment (spillerc-hpe) [reply RELEASE_NOTES.md]: I added the missing `cmn` network path.

File: .version [hunk 3]
```diff
@@ -1 +1 @@
-1.125.2
+1.13.12
```
Comment (johren-hpe) [inline .version:1]: This is confusing because the .version in the main branch right now is already 1.13.12 (not 1.125.2).

**Review by johren-hpe — APPROVED** [review]

**Pull request merged**